pkg/agent/hostagent.go

```diff
@@ -502,26 +490,34 @@ func (c *HostAgent) callRestartStatusAPI() error {
 // ListenForConfigChanges listens for configuration changes for the
 // agent on the Middleware backend and restarts the agent if configuration
 // has changed.
-func (c *HostAgent) ListenForConfigChanges(ctx context.Context) error {
+func (c *HostAgent) ListenForConfigChanges(ctx context.Context, errCh chan error) error {
+
+	// fetch the configuration for the first time
+	_, err := c.GetOtelConfig()
```

Comment:
In current setup, We are getting otel-config for the first time, in host agent.
Do we need to add this ?